test/fixtures/tw-import/output.js

```diff
@@ -1,3 +1,3 @@
 import { tw } from 'twind'
 const buttonClass = tw`bg-blue-500`
```

Comment:
Actually, after a second look, this call should be annotated too, shouldn't it?

Reply:
I didn't look at all code only at the changes. Sorry